lib/ButtonNew/ButtonIcon/ButtonIcon.js

```diff
@@ -34,7 +34,7 @@ function ButtonIcon({
 
   return (
     <ButtonBase className={classes} disabled={disabled} size={false} {...rest}>
-      <Icon name={name} title={iconTitle} />
+      <Icon name={name} title={iconTitle} defaultActiveColor={false} />
```

Comment:
This fixes a bug for ButtonIcons inside a parent with an `is-active` class forcing our icon to become blue when we may not want that to happen